.github/workflows/azure-static-web-apps.yml

```
@@ -0,0 +1,60 @@
+name: Deploy Azure Static Web Apps (Nuxt)
+
+on:
+  push:
+    branches: [ main ]
+    paths: [ 'src/web/**' ]
+  pull_request:
+    types: [opened, synchronize, reopened, closed]
+    branches: [ main ]
+    paths: [ 'src/web/**' ]
+
+jobs:
+  build_and_deploy_job:
+    if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.action != 'closed')
+    runs-on: ubuntu-latest
+    name: Build and Deploy Job
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: true
+          lfs: false
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          cache: 'npm'
+          cache-dependency-path: 'src/web/package-lock.json'
+
+      - name: Install dependencies for Web
+        working-directory: ./src/web
+        run: npm ci
+
+      - name: Build and Deploy
+        id: builddeploy
+        uses: Azure/static-web-apps-deploy@v1
+        with:
+          azure_static_web_apps_api_token: ${{ secrets.AZURE_STATIC_WEB_APPS_API_TOKEN }}
+          repo_token: ${{ secrets.GITHUB_TOKEN }} # Used for Github integrations (i.e. PR comments)
+          action: "upload"
+          ###### Repository/Build Configurations ######
+          app_location: "src/web" # App source code path
+          api_location: ".output/server" # Api source code path - optional
```

`api_location`はルートディレクトリからのパスであるべきだったので修正する
```suggestion
          api_location: "src/web/.output/server" # Api source code path - optional
```